README.rst

```diff
@@ -135,6 +135,36 @@ Currently only the Authorization Code authentication flow is supported.
      - ``.with_oidc()``
      -
 
+HTTPS Certificates
+~~~~~~~~~~~~~~~~~~
+
+The ``requests`` library does not use the system certificate store to verify TLS certificates, it uses the
```

Comment:
```suggestion
The ``requests`` library uses the ``certifi`` package to verify TLS certificates instead of a local system certificate store. These means only TLS certificates signed by a public CA can be verified by ``requests`` in its default configuration.
```